GDJS/Runtime/runtimebehavior.js

```diff
@@ -4,13 +4,19 @@
  * This project is released under the MIT License.
  */
 
+/**
+ * @typedef behaviorData
+ * @property {string} name The name of the behavior (for getting from an object (object.getBehavior) for example)
+ * @property {string} type The object Type. Used to get the constructor from gdjs
```

Comment:
The behavior type. Used by GDJS to find the proper behavior to construct.

Reply:
added